tests/test_AFC.py

```
@@ -4115,6 +4138,41 @@ def test_pushing_false_still_called_when_load_raises(self):
         assert events == [
             ('display', 'pushing', True), 'load_sequence', ('display', 'pushing', False)]
 
+    def test_load_to_gears_skips_purge_post_load_and_statistics(self):
+        afc, lane, events = self._make()
```

Ruff (0.16.2)

[warning] Unpacked variable `events` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)